driver.md

```diff
@@ -495,6 +566,19 @@ Here we check the other post-conditions associated with an EVM test.
     rule <k> check "transactions" : ("to"       : VALUE) => . ... </k> <txOrder> ListItem(TXID) ... </txOrder> <message> <msgID> TXID </msgID> <to>         VALUE </to>         ... </message>
     rule <k> check "transactions" : ("v"        : VALUE) => . ... </k> <txOrder> ListItem(TXID) ... </txOrder> <message> <msgID> TXID </msgID> <sigV>       VALUE </sigV>       ... </message>
     rule <k> check "transactions" : ("value"    : VALUE) => . ... </k> <txOrder> ListItem(TXID) ... </txOrder> <message> <msgID> TXID </msgID> <value>      VALUE </value>      ... </message>
+    rule <k> check "transactions" : ("chainId"  : VALUE) => . ... </k> <txOrder> ListItem(TXID) ... </txOrder> <message> <msgID> TXID </msgID> <txChainID>  VALUE </txChainID>  ... </message>
+    rule <k> check "transactions" : ("type"     : VALUE) => . ... </k> <txOrder> ListItem(TXID) ... </txOrder> <message> <msgID> TXID </msgID> <txType>     VALUE </txType>     ... </message>
+
+    syntax Bool ::= isInAccessListStorage ( Int , JSON )    [function]
+                  | isInAccessList ( Account , Int , JSON ) [function]
+ // ------------------------------------------------------------------
+    rule isInAccessList(_, _, [.JSONs]) => false
+    rule isInAccessList(ADDR, KEY, [[ACCT, [STRG:JSONs]],  REST]) => isInAccessListStorage (KEY, [STRG]) orBool isInAccessList(ADDR, KEY, [REST]) requires ADDR ==K #asAccount(#parseByteStackRaw(ACCT))
+    rule isInAccessList(ADDR, KEY, [[ACCT, [_STRG:JSONs]], REST]) => isInAccessList(ADDR, KEY, [REST])   requires notBool ADDR ==K #asAccount(#parseByteStackRaw(ACCT))
+
+    rule isInAccessListStorage(_, [.JSONs]) => false
+    rule isInAccessListStorage(KEY, [SKEY, _REST]) => true requires KEY ==Int #asWord(#parseByteStackRaw(SKEY))
+    rule isInAccessListStorage(KEY, [SKEY, REST])  => isInAccessListStorage(KEY, [REST]) requires  notBool KEY ==Int #asWord(#parseByteStackRaw(SKEY))
```

Comment:
Format these rules a bit better.

Also, I think we are better served by `#if_#then_#else_#fi` here actually. For both `isInAccessList` and `isInAccessListStorage`, we should use `#ite` in order to make it so we only need to call `#asWord(#parseByteStackRaw(SKEY))` once to determine which branch of the `#ite` to take, instead of calling it twice to check which rule applies.

Because this module is not used by the proofs at all, it won't hurt them to use `#ite`.